praw/models/reddit/redditor.py

```diff
@@ -187,9 +187,33 @@ def block(self):
 
             reddit.redditor("spez").block()
 
+        .. note::
+
+            Blocking a trusted user will remove that user from your trusted list.
+
+        .. seealso::
+
+            :meth:`~.trust`
```

Comment:
```suggestion
            :meth:`.trust`
```
The `~` isn't necessary in this instance.